warehouse/postgres/postgres.go

```diff
@@ -147,6 +147,7 @@ func (pg *HandleT) FetchSchema(warehouse warehouseutils.WarehouseT, namespace st
 	if err != nil {
 		return
 	}
+	defer pg.Db.Close()
```

Comment:
If these connections are created and destroyed inside this function, maybe we shall not keep the connection variable inside the HandleT?

Reply:
@snarkychef In this case yes, but we are using the HandleT in Process function where we need it across all receiver functions